### ISSUE TITLE
An error occurred while fetching shortcuts! (User 100 is locked or not running)

### ISSUE THREAD

**Issue by Qutition** [coderGtm/yantra-app-launcher#192]
**Describe the bug**
After restarting my phone this randomly start appearing, even if I do command reset or exit (I have it as default launcher)

**To Reproduce**
1. Restart your phone (hard restart)
2. Boot up

**Expected behavior**
My phone frozen, cause it had low percent (it's starting to glitch and freeze alot) I done a hard restart and I expected it to work normally, idk...

**Smartphone (please complete the following information):**
 - Device: [Honor x9b 5G 256Gb]
 - OS: [MagicOS 8.0 (Android 14)]
 - Version [9.7]

**Additional context**
Maybe that's not it, but when I do command list shortcuts it shows them.

- I will send screenshots rn, please wait

**Comment by Qutition** [comment]
https://github.com/user-attachments/assets/d64621a5-0734-4bc7-9f15-40d25af3e8f1


Its very long and vibrates when restarting/starting

**Comment by coderGtm** [comment]
Do you have Secure Folder on your device?

**Comment by Qutition** [comment]
What's is that?

**Comment by Qutition** [comment]
@coderGtm oh I have a thing called parallel space, its like a secury space for apps and files, and when I unlocking it, but clicking to fully shutdown the system its not giving the error! And apps in parrele space have PKG names and (work) after it

**Comment by Qutition** [comment]
I will upload later

**Comment by coderGtm** [comment]
I did not exactly understand the behaviour. When does the issue occur and when not?

**Comment by Qutition** [comment]
I https://github.com/user-attachments/assets/e2bda602-baf3-4a6d-b300-7909754ecf96

@coderGtm

**Comment by Qutition** [comment]
I'm just kinda busy rn so I typing fast, and I'm also not english. So my messages very delayed

**Comment by Qutition** [comment]
and the autocorrect is sucks

**Comment by coderGtm** [comment]
Okay got it.

This is like secure folder is samsung. I can't reproduce this issue as I don't have the test device but you can suggest a fix for it and we can code it!

**Comment by axolotl-git** [comment]
coderGtm I have a Samsung device (a galaxy J6), I can test it

**Comment by coderGtm** [comment]
Do you get this error?

**Comment by axolotl-git** [comment]
No, I have never had this error and I just did the test, but I have never activated the Samsung secure folder function, so I deduce that if it is deactivated it is not bugged. Give me a moment and I'll try to activate the secure folder and try to reproduce the bug

**Comment by axolotl-git** [comment]
Sorry but I can't do the test with the secure folder (Secure Area) active, the Samsung app itself is bugged lmao

**Comment by Anready** [comment]
i also have this error sometimes but I can't understand how i can reproduce it

**Comment by coderGtm** [comment]
If someone reproduces it then please share insights
